Correct .NET Framework bug documentation for MMM format specifier genitive behavior (#48061)
dotnet/docs: docs/standard/base-types/custom-date-and-time-format-strings.md

| "mm" | The minute, from 00 to 59.<br /><br /> More information: [The "mm" Custom Format Specifier](#mmSpecifier). | 2009-06-15T01:09:30 -> 09<br /><br /> 2009-06-15T01:45:30 -> 45 |
| "M" | The month, from 1 to 12.<br /><br /> More information: [The "M" Custom Format Specifier](#M_Specifier). | 2009-06-15T13:45:30 -> 6 |
| "MM" | The month, from 01 to 12.<br /><br /> More information: [The "MM" Custom Format Specifier](#MM_Specifier). | 2009-06-15T13:45:30 -> 06 |

> | "MMM" | The abbreviated name of the month. When used with "d" or "dd" specifiers, uses genitive names (note: this is a known bug in .NET Framework that was fixed in .NET).<br /><br /> More information: [The "MMM" Custom Format Specifier](#MMM_Specifier). | 2009-06-15T13:45:30 -> Jun (en-US)<br /><br /> 2009-06-15T13:45:30 -> juin (fr-FR)<br /><br /> 2009-06-15T13:45:30 -> Jun (zu-ZA) |

Reviewer: >(note: this is a known bug in .NET Framework that was fixed in .NET) [](http://example.com/codeflow?start=104&length=68)

Instead of saying `this is a known bug in .NET Framework that was fixed in .NET` we say `This was a known issue in the .NET Framework that has been resolved in .NET.`

### <a name="MMM_Specifier"></a> The "MMM" custom format specifier

> The "MMM" custom format specifier represents the abbreviated name of the month. The localized abbreviated name of the month is retrieved from the <xref:System.Globalization.DateTimeFormatInfo.AbbreviatedMonthNames%2A?displayProperty=nameWithType> property of the current or specified culture. If there is a "d" or "dd" custom format specifier in the custom format string, it is retrieved from the <xref:System.Globalization.DateTimeFormatInfo.AbbreviatedMonthGenitiveNames%2A?displayProperty=nameWithType> property instead. Note that this genitive behavior for "MMM" is a known bug in .NET Framework that was fixed in .NET.

Reviewer: > Note that this genitive behavior for "MMM" is a known bug in .NET Framework that was fixed in .NET [](http://example.com/codeflow?start=523&length=99)

same here.

Reviewer 2: ```suggestion
The "MMM" custom format specifier represents the abbreviated name of the month. The localized abbreviated name of the month is retrieved from the <xref:System.Globalization.DateTimeFormatInfo.AbbreviatedMonthNames%2A?displayProperty=nameWithType> property of the current or specified culture. If there is a "d" or "dd" custom format specifier in the custom format string, it is retrieved from the <xref:System.Globalization.DateTimeFormatInfo.AbbreviatedMonthGenitiveNames%2A?displayProperty=nameWithType> property instead, except in .NET Framework. (This is known issue that's fixed in .NET.)
```
